plugins/inputs/fluentd/fluentd.go

```diff
@@ -0,0 +1,195 @@
+package fluentd
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"reflect"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+const (
+	measurement  = "fluentd"
+	description  = "Read metrics exposed by fluentd in_monitor plugin"
+	sampleConfig = `
+	## This plugin only reads information exposed by fluentd using /api/plugins.json.
+	## Tested using 'fluentd' version '0.14.9'
```

Comment:
Move this comment about version in with the tests, so we can see the version you generated the sample data from.  If you know the plugin will not work with any specific versions of fluentd, then we can put that in the README.